clients/src/main/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumer.java

```diff
@@ -214,10 +213,11 @@ private void process(final ConsumerRebalanceListenerCallbackNeededEvent event) {
     private final ApplicationEventHandler applicationEventHandler;
     private final Time time;
     private final AtomicReference<Optional<ConsumerGroupMetadata>> groupMetadata = new AtomicReference<>(Optional.empty());
-    private final KafkaConsumerMetrics kafkaConsumerMetrics;
+    private final AsyncConsumerMetrics asyncConsumerMetrics;
```

Comment:
I'd prefer it be left as `kafkaConsumerMetrics`. Calling out the distinction in the variable name isn't really adding anything (IMO).